releng/releng-target/xtext-core.target.target

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?pde version="3.8"?>
-<target name="org.eclipse.xtext.core.target" sequenceNumber="0">
+<target name="org.eclipse.xtext.helios.target" sequenceNumber="0">
```

Comment:
this change is produced by the generateP2Build task and should be reverted

Reply:
will d you that or should i?
we fix the generate p2 build task then

Reply:
OK, we'll put this on our list

Reply:
Tracked in https://github.com/eclipse/xtext/issues/1220